### DISCUSSION TITLE
2007/01/26/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-4-%E5%8A%A8%E6%80%81%E8%BF%9E%E6%8E%A5%E5%BA%93%E5%87%BD%E6%95%B0%E7%9A%84%E7%BA%BF%E7%A8%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#141] [Announcements]
# 2007/01/26/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-4-%E5%8A%A8%E6%80%81%E8%BF%9E%E6%8E%A5%E5%BA%93%E5%87%BD%E6%95%B0%E7%9A%84%E7%BA%BF%E7%A8%8B

四、动态连接库函数的线程

https://blog.qizhen.xyz/2007/01/26/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-4-%E5%8A%A8%E6%80%81%E8%BF%9E%E6%8E%A5%E5%BA%93%E5%87%BD%E6%95%B0%E7%9A%84%E7%BA%BF%E7%A8%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

捷周
2007年02月6日 14:18 编辑
hi,
  我用Labwindow 编写了一个读文件的动态库, 即向动态库传递文件路径及文件名和某特定字符串,然后通个三个参数返回读到的值, 在labVIEW里调用该动态库,结果返回值老是显示打开文件失败, 不知错误出现在那里,另外在LabVIEW里如何调试确定传到动态库的参数是符合函数参数格式的呢?该函数在Labwindow里调试没有问题.请大侠指点迷津,不胜感激!

回复
